sys/include/net/emcute.h

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_emcute emCute (MQTT-SN)
```

Comment:
Would prefer the name "MQTT-SN client (emCute)" so it jumps into the eyes of newbies ;-).